### PR TITLE
cypress pack initial commit

### PR DESCRIPTION
**Pull request by khalidelaggan**

### CODE REVIEW

File: f2/cypress/integration/walkthough.spec.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
+describe('Entire form form filling testing...', function () {
+    /**
+    ----------------------------------------------------
+    Manual filling workflow route table configurations
+    ----------------------------------------------------
+
```
Comment (lmcbhvu) [inline f2/cypress/integration/walkthough.spec.js]: I don't like this, please change it .... RCMP test :)

**Comment by khalidelaggan** [comment]
@lmcbhvu  let's aim for tomorrow to get this PR ready to be 🚢

**Comment by khalidelaggan** [comment]
Just a quick critical update for  @davidheerema @sastels @ngosset - >
Hong and I are aiming to have this ready by the end of the day.

File: .github/workflows/test_f2.yml [hunk 2]
```diff
@@ -23,6 +23,13 @@ jobs:
           npm run build --if-present
         env:
           CI: true
+      - name: Run Cypress end-to-end
+        working-directory: f2
+        run: npm run cypress:cli
+        env:
+          CI: true
+          NODE_ENV: testing
+          A11Y_TRACKER_KEY: ${{ secrets.A11Y_TRACKER_KEY }}
```
Comment (khalidelaggan) [inline .github/workflows/test_f2.yml]: Would need to figure out adding this to the secret

**Comment by khalidelaggan** [comment]
@sastels @maxneuvians ready to review and address any comments 👍

**Comment by sastels** [comment]
Running locally with `npm run cypress` then clicking "run all specs" I get two test failures

image

**Comment by sastels** [comment]
The file `cypress/videos/examples/aliasing.spec.js.mp4` gets deleted when I run `npm run cypress:cli` - should it be in there?

**Comment by khalidelaggan** [comment]
> Running locally with `npm run cypress` then clicking "run all specs" I get two test failures
> 
> image
> 
> `npm run cypress:cli` succeeds though 🎉

Yes! as the test that Hong written needs to be adjusted slightly to the changes in Staging, it will be a perfect exercise to show the team ins the outs of tests workflow that aren't aligned with code changes :)

**Comment by khalidelaggan** [comment]
> Looks good! will be nice to get this in the repo add a few more smoke tests 👍
> 
> Just swap out those images and delete that video and we're good to go!

Excellent! more tests to be merged today for sure!

**Comment by khalidelaggan** [comment]
Happy this is being merged, so we can broaden writing tests with more participant now.

**Pull request merged**